tests/unit_and_functional/test_k8s_service_deployments.py

```diff
@@ -471,17 +472,82 @@ def test_list_service_stage_deployments_returns_service_stage_info(
                     )
                 )
                 deployment_info = list_service_stage_deployments(service_namespace)
+                mock_k8s_apps_api().list_namespaced_deployment.assert_called_with(
+                    namespace=service_namespace, label_selector="app=bodywork"
+                )
                 assert service_name in deployment_info.keys()
                 assert deployment_info[service_name]["service_url"] == service_url
                 assert deployment_info[service_name]["service_port"] == service_port
-                assert deployment_info[service_name]["service_exposed"] == "true"
+                assert deployment_info[service_name]["service_exposed"] is True
                 assert deployment_info[service_name]["available_replicas"] == 1
                 assert deployment_info[service_name]["unavailable_replicas"] == 0
                 assert (
                     deployment_info[service_name]["git_branch"] == "project_repo_branch"
                 )  # noqa
                 assert deployment_info[service_name]["git_url"] == "project_repo_url"
-                assert deployment_info[service_name]["has_ingress"] == "true"
+                assert deployment_info[service_name]["has_ingress"] is True
+
+
+@patch("kubernetes.client.AppsV1Api")
+def test_list_service_stage_deployments_returns_all_services_on_cluster(
+    mock_k8s_apps_api: MagicMock,
+    service_stage_deployment_object: kubernetes.client.V1Deployment,
+):
+    service_namespace = service_stage_deployment_object.metadata.namespace
+    service_name = service_stage_deployment_object.metadata.name
+
+    service_stage_deployment_object.status = kubernetes.client.V1DeploymentStatus(
+        available_replicas=1, unavailable_replicas=None
+    )
+
+    service_stage_deployment_object2 = copy.deepcopy(service_stage_deployment_object)
+    service_stage_deployment_object2.metadata.name = "deployment two"
```

Comment:
`deployment-two` - I think whitespace is an invalid k8s names. Not that is matters, but...

Reply:
Done!